tst/EnergyPlus/unit/PlantHeatExchangerFluidToFluid.unit.cc

```diff
@@ -2481,6 +2481,14 @@ TEST_F(EnergyPlusFixture, PlantHXControl_CoolingSetpointOnOffWithComponentOverri
     PlantHeatExchangerFluidToFluid::InitFluidHeatExchanger(1, 1);
 
     EXPECT_NEAR(DataPlant::PlantLoop(1).LoopSide(2).Branch(2).Comp(1).FreeCoolCntrlMinCntrlTemp, 9.5, 0.001);
+
+    // Force Effectiveness = 1 when -NTU < EXP_LowerLimit #6516
+    PlantHeatExchangerFluidToFluid::FluidHX(1).HeatExchangeModelType = PlantHeatExchangerFluidToFluid::CrossFlowBothMixed;
+    PlantHeatExchangerFluidToFluid::FluidHX(1).UA = 9000.0;
+
+    PlantHeatExchangerFluidToFluid::CalcFluidHeatExchanger(1, 0.1, 0.1);
```

Comment:
This is the change that already went in with the other PR, right?  I'll have to verify.

Reply:
@Myoldmopar Yes. The change was in #6516.